packages/bits/spec/components/menu/menu.e2e.ts

```diff
@@ -65,16 +68,13 @@ describe("USERCONTROL Menu", () => {
             });
 
             it("should NOT open and close menu by ENTER key if focused on toggle", async () => {
-                await menu.toggleMenu();
-                await menu.toggleMenu();
```

Comment:
It seems like these might be needed to ensure that the correct menu has focus before hitting the enter key.